sdk/identity/src/token_credentials/azure_cli_credentials.rs

```diff
@@ -47,9 +47,15 @@ struct CliTokenResponse {
 }
 
 /// Enables authentication to Azure Active Directory using Azure CLI to obtain an access token.
+#[derive(Default)]
 pub struct AzureCliCredential;
```

Comment:
FYI: users can still use the `AzureCliCredential` syntax for instantiating this. This means that adding any fields is a breaking change. We may want to consider adding a private field just to prevent this.

```rust
struct AzureCliCredential {
    _private: ()
}
````